recipes/kedro-docker/recipe.yaml

```diff
@@ -0,0 +1,53 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/prefix-dev/recipe-format/main/schema.json
+schema_version: 1
+
+context:
+  name: kedro-docker
+  version: "0.6.2"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/kedro_docker-${{ version }}.tar.gz
+  sha256: c95d04c89ad7725df6ac486800949b0b580e96e57f441f107792db97bc67c3f9
+
+build:
+  number: 0
+  noarch: python
+  script: ${{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation --disable-pip-version-check
+
+requirements:
+  host:
+    - python ${{ python_min }}.*
+    - setuptools >=61.2
+    - pip
+  run:
+    - python >=${{ python_min }},<4.0
+    - anyconfig >=0.10.0,<0.11.dev0
+    - kedro >=0.16.0
+    - semver >=2.10,<3.dev0
+    - click >=4.0,<8.2.0
+
+tests:
```

Comment:
Can you add the pip check test?

Reply:
will do - @chrisburr pointed me to https://rattler.build/latest/reference/recipe_file/#tests-section
https://github.com/conda-forge/staged-recipes/pull/30136#discussion_r2106687972
where is shows pip_check: true  # this is the default
So I assumed, skipping it meant pip_check was run..

Reply:
Ah. I did not know it was default! No problem then.

Reply:
I'll add pip_check: true so that its obvious..